src/scripts/processsolar.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3.8
+#!/usr/local/bin/python
```

Comment:
should it not be `python3` at least or is this code python2? :fearful:

Reply:
python3.8 is used,  but /usr/bin/python3.8 is not available, below you see output from the running container.

root@1fc73e3771a3:/app# python
Python 3.8.5 (default, Sep  1 2020, 18:44:24) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> exit()
root@1fc73e3771a3:/app# which python
/usr/local/bin/python
root@1fc73e3771a3:/app#

Reply:
ah then everything is fine.